python/triton/language/extra/hip/libdevice.py

```diff
@@ -0,0 +1,433 @@
+from triton.language import core
+
+
+@core.extern
+def abs(arg0, _builder=None):
+    return core.extern_elementwise(
+        "", "", [arg0], {
+            (core.dtype("int32"), ): ("__ocml_trition_abs", core.dtype("int32")),
+            (core.dtype("int64"), ): ("__ocml_trition_abs", core.dtype("int64")),
+            (core.dtype("fp32"), ): ("__ocml_trition_abs", core.dtype("fp32")),
```

Comment:
Given we control the symbol names here. Why not spliting them into `__triton_hip_iabs` and `__triton_hip_fabs` so that it's simpler in later conversion matching?

Reply:
It is a good idea. I like it.

Done